EC-84: Restructure eZ Commerce doc, part 2 (#2)
ezsystems/commerce-documentation: docs/guide/product_description/manage_products_prices_and_stock.md

## Flexible categories

> A category contains a standard set of fields. A category is a Content Type and can be extended with additional Fields.

Reviewer: ```suggestion
A category contains a standard set of Fields. A category is a Content Type and can be extended with additional Fields.
```

![](img/image2018-4-4_19-14-10.png)

## Product - product texts

> eZ Commerce offers a set of product Fields. Since a product is built using the flexible Content Type system it can be extended with new Fields easily.

Reviewer: ```suggestion
eZ Commerce offers a set of product Fields. Since a product is built using the flexible Content Type system it can be easily extended with new Fields.
```

## Product - Variants

eZ Commerce offers one or two level variants. A variant can be added directly on the product view.

> There are 3 preconfigured variant types: Color, Size, Color & Size

Reviewer: ```suggestion
There are three preconfigured variant types: Color, Size, Color and Size
```

## Product prices

> The price management enables setting up prices manually. A price can be setup per SKU and SKU/variant. Each price can contain an offer price and a base price. If an offer price is set it will be displayed in the shop as

Reviewer: ```suggestion
The price management enables setting up prices manually. A price can be setup per SKU and SKU/variant. Each price can contain an offer price and a base price. If an offer price is set it will be displayed in the shop as:
```

~~Old price: 10.00 €~~ New price: 9.80 €

> In addition prices can vary per customer group. There are 3 customer groups set up per default. 

Reviewer: ```suggestion
In addition, prices can vary per customer group. There are three customer groups set up per default. 
```

![](img/Price_management.png)

Currencies in eZ Commerce:

The currency is configured per country in the [configuration settings](ecommerce_administration.md).

- if a product has a price for a product and a currency this price will be displayed in the shop

> - if no price for a currency is set in a shop eZ Commerce offers 2 options, these can be set in the configuration per shop.

Reviewer: ```suggestion
- if no price for a currency is set in a shop, eZ Commerce offers two options, these can be set in the configuration per shop.
```
